balances/src/lib.rs

```diff
@@ -164,6 +164,8 @@ pub mod pallet {
 		Endowed { cid: CommunityIdentifier, who: T::AccountId, balance: BalanceType },
 		/// Token transfer success `[community_id, from, to, amount]`
 		Transferred(CommunityIdentifier, T::AccountId, T::AccountId, BalanceType),
+		/// Token issuance success `[community_id, beneficiary, amount]`
+		Issued(CommunityIdentifier, T::AccountId, BalanceType),
```

Comment:
In the future, I'd like to have event enums named fields, it's much clearer. But this should be a separate task.